src/styles/forms/_inputs.scss

```diff
@@ -84,23 +84,23 @@
     border: none;
     border-radius: 3px;
     box-sizing: border-box;
-    color: var(--KInputColor, var(--black-70, color(black-70)));
+    color: var(--KInputColor, var(--black-70, rgba(0, 0, 0, 0.7)));
     display: block;
-    font-size: var(--type-md, type(md));
+    font-size: var(--type-md, 16px);
     font-weight: 400;
```

Comment:
question: are you not replacing all of these type of values as well?

Reply:
Right but why not use a token..?

Reply:
Ah sorry, I misunderstood your question. Styles in this specific file will be moved to the component/mixins soon - that's when I was planning to go though all CSS rules line-by-line (you can notice that this file has a lot more non-tokenized rules). That's why didn't bother inserting the token here initially. I fixed it now but going further ignore hardcoded/non-token values in these standalone files.

Reply:
Ah ok makes sense; thanks for the context